stablehlo/dialect/StablehloOps.cpp

```diff
@@ -138,9 +138,13 @@ void CollectivePermuteOp::build(OpBuilder& odsBuilder, OperationState& odsState,
 //===----------------------------------------------------------------------===//
 
 LogicalResult ReduceScatterOp::verify() {
+  int64_t channelId = 0;
```

Comment:
`ChannelId channelId = 0`

Reply:
Similar comment to https://github.com/openxla/stablehlo/pull/1727#discussion_r1298927332. Resolving .